dora/integration/jnifuse/fs/src/main/java/alluxio/jnifuse/AbstractFuseFileSystem.java

```diff
@@ -280,6 +280,15 @@ public int mkdirCallback(String path, long mode) {
     }
   }
 
+  public int renameCallback(String oldPath, String newPath) {
+    try {
+      return rename(oldPath, newPath, 0);
+    } catch (Exception e) {
+      LOG.error("Failed to rename {}, newPath {}: ", oldPath, newPath, e);
+      return -ErrorCodes.EIO();
+    }
+  }
```

Comment:
Prefer `return renameCallback(oldPath, newPath, 0);` to remove code duplication

Reply:
Good catch. Updated.